webview/src/experiments/components/table/styles.module.scss

```diff
@@ -56,6 +56,29 @@ $bullet-size: calc(var(--design-unit) * 4px);
   }
 }
 
+%cellBorderLeft {
```

Comment:
By having all cells, both header and body, use the same `:before` method to create the left border, the borders become perfectly aligned.